src/pages/About/MissionAndPurpose/MissionAndPurpose.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+// import PropTypes from 'prop-types'
+import Container from '../../../components/Container/Container.js'
+
+import './MissionAndPurpose.scss';
+
+export const AboutMissionTemplate = ({
+    section_uno,
+    section_two,
+    section_three,
+    }) => (
+        <div className="about-mission-purpose--content" style= {{textAlign:"left"}}>
+            <h1 className="title section-one__title">{section_uno.title}</h1>
+            <div className="body section-one__body" dangerouslySetInnerHTML = {createMarkup(section_uno.body)}/>
+            <h1 className="title section-two__title">{section_two.title}</h1>
+            <div className="body section-two__body" dangerouslySetInnerHTML = {createMarkup(section_two.body)}/>
+            <h1 className="title section-three__title">{section_three.title}</h1>
+            <div className="body section-three__body" dangerouslySetInnerHTML = {createMarkup(section_three.body)}/>
+        </div>
+    )
+
+function createMarkup(body) {
+  return {__html: body};
+}
+
+const aboutMission = ({ data }) =>{
+
+    if (!data) return null;
+
+    return(
+        <Container>
+            <AboutMissionTemplate
+                section_uno = {data.mission_and_purpose.section_one} 
```

Comment:
```suggestion
                section_one = {data.mission_and_purpose.section_one} 
```
The spacing is a little much here. Try setting your indents to two.